pkgs/games/fltrator/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, unzip, fltk, which, libjpeg }:
+
+stdenv.mkDerivation rec {
+  name = "fltrator-${version}";
+  version = "2.3";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/fltrator/fltrator-${version}-code.zip";
+    sha256 = "125aqq1sfrm0c9cm6gyylwdmc8xrb0rjf563xvw7q28sdbl6ayp7";
+  };
+
+  buildInputs = [ fltk unzip which libjpeg ];
```

Comment:
Dependencies that are run during build should go to `nativeBuildInputs ` (unzip, which)